### PR TITLE
Hud branch

### PR DESCRIPTION
**Pull request by SeanAbnerNash**

### CODE REVIEW

File: ARGO/include/Utilities.h [hunk 1]
```diff
@@ -70,7 +70,13 @@ namespace Utilities
 
 	//Flow field values
 	const int MAX_FLOW_FIELD_WEIGHT = 20;
+	const float RADIANS_TO_DEGREES = 57.2958;
```
Comment (PrzemekTomczyk) [inline ARGO/include/Utilities.h]: not sure if you added this but its showing up here, glm has a function that converts deg/rad to rad/deg

File: ARGO/src/PickUpManager.cpp [hunk 2]
```diff
@@ -17,7 +17,7 @@ void PickUpManager::init(SDL_Renderer* t_renderer)
 	PickUpFactory* factory = new PickUpFactory(m_renderer);
 	for (int i = 0; i < PICKUP_POOL_SIZE; i++)
 	{
-		factory->createDrop(1, m_pickUps[i]);
+		factory->createDrop(2, m_pickUps[i]);
```
Comment (PrzemekTomczyk) [inline ARGO/src/PickUpManager.cpp:20]: why the change here?

Comment (SeanAbnerNash) [reply ARGO/src/PickUpManager.cpp:20]: Health Is Implemented and affects the player and Hud. This is here to demonstrate that both systems are working.

File: ARGO/include/Utilities.h (hunk 1 above)
Comment (SeanAbnerNash) [reply ARGO/include/Utilities.h]: Removed.
